packages/terra-paginator/src/_PaginatorButton.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { disableFocusActiveStyles, disableFocusOnBlur, enableFocusActiveStyles } from './_paginationUtils';
+import styles from './Paginator.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  children: PropTypes.node,
+  /**
+   * Callback function triggered when clicked.
+   */
+  onClick: PropTypes.func,
+};
+
+class PaginatorButton extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleFocus = this.handleFocus.bind(this);
+    this.handleMouseDown = this.handleMouseDown.bind(this);
+
+    this.shouldShowFocus = true;
+  }
+
+  handleFocus(event) {
+    if (this.shouldShowFocus) event.currentTarget.setAttribute('data-focus-styles-enabled', 'true');
+  }
+
+  handleMouseDown() {
+    // Prevent button from showing focus styles when clicked
+    this.shouldShowFocus = false;
+    // Wait until after onFocus has been triggered on browsers where it will get triggered for click
+    setTimeout(() => { this.shouldShowFocus = true; });
+  }
+
+  render() {
+    const {
+      children,
+      onClick,
+      ...customProps
```

Comment:
This is a private component, there's really no need to allow for custom props.

Reply:
Changed: [243ad46](https://github.com/cerner/terra-core/pull/2686/commits/243ad46953a80fa9e1f6fa19c0924da3ab009538)